hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/protocol/StorageContainerLocationProtocol.java

```diff
@@ -230,4 +230,5 @@ Pipeline createReplicationPipeline(HddsProtos.ReplicationType type,
    */
   boolean getReplicationManagerStatus() throws IOException;
 
+  List<String> getScmRatisStatus() throws IOException;
```

Comment:
Does this API belongs to `StorageContainerLocationProtocol`?

Reply:
Any way to merge it with GetSCMInfo?

Reply:
Pushed a new commit to merge this logic into `GetScmInfo`